enhancements/ingress/host-network-bind-options.md

```diff
@@ -0,0 +1,266 @@
+        // http connections. Setting this field is generally not recommended. However in
+        // HostNetwork strategy, default http 80 port might be occupied by other processess
+        //
+        // +kubebuilder:validation:Optional
+        // +kubebuilder:validation:Minimum=1
+        // +kubebuilder:validation:Maximum=30000
+        // +kubebuilder:default:=80
+        // +optional
+        HTTP int32 `json:"http,omitempty"`
+        // https defines the port number which HAProxy process binds for
+        // https connections. Setting this field is generally not recommended. However in
+        // HostNetwork strategy, default https 443 port might be occupied by other processess
+        //
+        // +kubebuilder:validation:Optional
+        // +kubebuilder:validation:Minimum=1
+        // +kubebuilder:validation:Maximum=30000
+        // +kubebuilder:default:=443
+        // +optional
+        HTTPS int32 `json:"https,omitempty"`
+        // stats is the port number which HAProxy process binds
+        // to expose statistics on it. Setting this field is generally not recommended.
+        // However in HostNetwork strategy, default stats port 1936 might
+        // be occupied by other processess
+        //
+        // +kubebuilder:validation:Optional
+        // +kubebuilder:validation:Minimum=1
+        // +kubebuilder:validation:Maximum=30000
+        // +kubebuilder:default:=1936
+        // +optional
+        Stats int32 `json:"stats,omitempty"`
+    }
+    ```
+
+The following example configures two ingress controllers with the HostNetwork strategy that can run on the same nodes without port conflicts:
+
+    ```yaml
+    ---
+    apiVersion: operator.openshift.io/v1
+    kind: IngressController
+    metadata:
+    name: example-1
+    namespace: openshift-ingress-operator
+    spec:
+    endpointPublishingStrategy:
+        type: HostNetwork
+    ---
+    apiVersion: operator.openshift.io/v1
+    kind: IngressController
+    metadata:
+    name: example-2
+    namespace: openshift-ingress-operator
+    spec:
+    endpointPublishingStrategy:
+        type: HostNetwork
+        hostNetwork:
+            bindingOptions:
+                ports:
+                    http: 11080
+                    https: 11443
+                    stats: 1937
+    ```
+### Validation
+
+Setting `spec.endpointPublishingStrategy.type: HostNetwork` and ignoring
+`spec.endpointPublishingStrategy.hostNetwork` or
+`spec.endpointPublishingStrategy.hostNetwork.bindingOptions` is valid and implies
+the default behavior, which is to use the default ports to bind to. 
+
+The API validates
+that any value provided for the ports defined in
+`spec.endpointPublishingStrategy.hostNetwork.bindingOptions` is greater than `1`, and lower than `30000` (to avoid conflicts with 30000-32767 nodePort services range).
+
+### User Stories
+
+#### As a cluster administrator, I need to set binding ports for the IngressController
```

Comment:
maybe add so that I can deploy multiple ingress controllers on the same node for the HostNetwork strategy